content/en/core/releases/3.17.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "3.17.0 release notes"
+linkTitle: "3.17.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.17.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### Labels of action buttons are now available in small screens
+
+A number of commonly used phones have a horizontal resolution of 720 with a default dp of 360. We have enabled the labels of action buttons for those small screens to make actions easier to understand.
```

Comment:
This is probably overly technical for the release notes. Can we just say, all devices now show the label?